grails-app/views/location/list.gsp

```diff
@@ -122,7 +122,7 @@
                                                 </g:else>
 
                                             </td>
-                                            <g:each var="activity" in="${org.pih.warehouse.core.ActivityCode.list()}">
+                                            <g:each var="activity" in="${grailsApplication.config.openboxes.autosave.enabled ? ActivityCode.listWithAutosave() : ActivityCode.list()}">
```

Comment:
For my understanding, if we disable it for openboxes, any location shouldn't have option to use it. No excuses 😅

Reply:
I know you shouldn't be able then to set it, but my point is, if the instance has it enabled and then you disable it, while a location has this supported activity, if it should be hidden in the location view page (even though it's still set in the database), or it should be visible, but somehow crossed out so that user would understand, that it's disabled, but "physically" the location has it.